src/opm/parser/eclipse/EclipseState/Runspec.cpp

```diff
@@ -472,11 +472,13 @@ Nupcol::Nupcol(const Deck& deck) :
         const auto& min_item = runspecSection.getKeyword<ParserKeywords::MINNPCOL>().getRecord(0).getItem<ParserKeywords::MINNPCOL::VALUE>();
         this->min_nupcol = min_item.get<int>(0);
     }
+    // make sure the default value is above the minimum
+    Nupcol::update(Nupcol::value());
```

Comment:
Sorry I am being anal here; but I want the default constructor to be rewritten as:
```C++
Nupcol()
   : min_nupcol( ParserKeywords::MINNPCOL::VALUE::defaultValue) 
{
      this->update( ParserKeywords::NUPCOL::NUM_ITER::defaultValue);
}
```